### DISCUSSION TITLE
Cooperation in education

### DISCUSSION THREAD

**Discussion by opx14** [Cyfrin/foundry-full-course-cu#1775] [General]
Hello friends. I am looking to create an environment to learn solidity programming. In most cases, I have seen if it is a team or an environment is created where everyone works in the same field. The effort and passion for that training increases. We can do this training course together.  let's spend and then cooperate. If you are interested, send a message to this ID in Telegram. @Moh3bak

**Comment by EngrPips** [comment]
Awesome, I wish you the best in your search for a buddy @opx14

**Reply by opx14** [reply]
Thanks ❤️

**Reply by EngrPips** [reply]
Anytime friend.
